interface-definitions/firewall.xml.in

```diff
@@ -205,6 +205,63 @@
               </leafNode>
             </children>
           </tagNode>
+          <tagNode name="external-list">
+            <properties>
+              <help>Firewall external lists</help>
+              <constraint>
+                <regex>[a-zA-Z0-9][\w\-\.]*</regex>
+              </constraint>
+            </properties>
+            <children>
+              <leafNode name="url">
+                <properties>
+                  <help>URL to fetch the populated list (one address/network per line)</help>
+                  <valueHelp>
+                    <format>url</format>
+                    <description>Remote URL to the list text file</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>path</format>
+                    <description>Local path to the list text file</description>
+                  </valueHelp>
+                  <constraint>
+                    <regex>(https?|ftp):\/\/([a-zA-Z0-9]\.){2,}*</regex>
+                    <validator name="file-exists"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="crontab-spec">
```

Comment:
As there is a dedicated task-scheduler (crontab CLI node) this somehow does not feel right.

Maybe an idea is to have predefined crontab tasks a user can activate via the task-scheduler subsystem to refresh this list?

Reply:
Not sure if it helps, but I removed the duplicate definitions and used an include for interval and crontab-spec configuration.  Now that I understand how the configuration definitions work this is much cleaner.

Reply:
+1 on making this an included file - thanks for knowing this design.

I really wonder if a "user entered" cron job is really required. Why not make this refresh every 5,10 or 15 minutes? Is it bad to hardcode this?